iam/api-client/service_account_keys.py

```diff
@@ -50,7 +51,11 @@ def create_key(service_account_email):
         name='projects/-/serviceAccounts/' + service_account_email, body={}
         ).execute()
 
-    print('Created key: ' + key['name'])
+    # The privateKeyData field contains the base64-encoded service account key
+    # in JSON format. Save this key to a secure location. You cannot download it
```

Comment:
To avoid the lint error, can we instead comment out line 57 and then suggest "Uncomment the next line and save this key to a secure location"

Reply:
If we comment the line 57, then the variable in line 50 will not be used which would throw a lint error :(
Is there any other way around this?

Reply:
Could you try:
```suggestion
    print(f'Created json key: {json_key_file}')
```
or something similar to print that the key has been made (or other properties of the json key file that's more appropriate), that way we know that the key was created for sure?

Reply:
IAM tech writer here. We shouldn't print the entire key file; it contains a private key, which is sensitive data.

Reply:
That sounds fair. I'll leave it to @Sita04 to figure out what would be the most appropriate way to use that key without revealing sensitive data.

Reply:
I commented out the `json_key_file` and instead used the key status param `key['disabled']` to validate if the key was created.
@hegemonic @dandhlee PTAL